### PR TITLE
[DX-613] Update okta-dcr.md

### PR DESCRIPTION
**Pull request by firaaasss**
Updated the doc to reflect the correct information - application type is "web" not "service". Important to mention that it this can be changed on the EDP.

### For internal users - please add a Jira DX PR ticket to the subject!
---
## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

**Comment by letzya** [comment]
@Firaasss thanks for the ticket, please get a peer review and label it with `now`

**Review by Monica-Cedeno — APPROVED** [review]
Looks good! Great work!

File: tyk-docs/content/tyk-developer-portal/tyk-portal-classic/okta-dcr.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ This value is found in your well-known discovery document as `registration_endpo
...
+It’s important to note that OKTA’s DCR endpoint supports a parameter called `"application_type"`, the application types aren’t standard across all IDPs, while the initial specification mentions `"native"` or `"web"` types, some IDPs implement their own. In the current implementation Tyk supports the usage of the `"web"` application type which is necessary in supporting the client credentials flow that’s described in this guide, as well as others, this is set automatically when OKTA is set as the provider. Currently, the ability to change the application type is available with the Enterprise Developer Portal.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-developer-portal/tyk-portal-classic/okta-dcr.md:134]:
```suggestion
It’s important to note that OKTA’s DCR endpoint supports a parameter called `"application_type"`. The application types aren’t standard across all IDPs. While the initial specification mentions `"native"` or `"web"` types, some IDPs implement their own. In the current implementation Tyk supports the usage of the `"web"` application type, which is necessary in supporting the client credentials flow that’s described in this guide. This is set automatically when OKTA is configured as the provider. Currently, the ability to change the application type is available with the Enterprise Developer Portal.
```

Comment (dcs3spp) [reply tyk-docs/content/tyk-developer-portal/tyk-portal-classic/okta-dcr.md:134]: Hi @Firaasss Thanks for raising the PR in relation to OKTA based on the discussion in this [thread](https://tyktech.slack.com/archives/C51P42LLT/p1689608929514729). Should we add a link to how to configure application type in Enterprise Developer Portal as given in the [thread](https://tyktech.slack.com/archives/C51P42LLT/p1689608929514729)? 

This page is for classic portal. Has Tyk Classic Portal being tested with `web` application_type?  After reading the [thread](https://tyktech.slack.com/archives/C51P42LLT/p1689608929514729), `web` application_type is not supported on classic portal with OKTA? If this is the case should a sentence be added to explain this?

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Hi @Firaasss Thanks for raising the PR in relation to OKTA based on the discussion in this [thread](https://tyktech.slack.com/archives/C51P42LLT/p1689608929514729). Should we add a link to how to configure application type in Enterprise Developer Portal as given in the [thread](https://tyktech.slack.com/archives/C51P42LLT/p1689608929514729)? 

This page is for classic portal. Has Tyk Classic Portal being tested with `web` application_type?  After reading the [thread](https://tyktech.slack.com/archives/C51P42LLT/p1689608929514729), `web` application_type is not supported on classic portal with OKTA? If this is the case should a sentence be added to explain this?

**Comment by letzya** [comment]
@Firaasss can I please check what's the status of this PR?

**Comment by firaaasss** [comment]
> @Firaasss can I please check what's the status of this PR?

@letzya This has been ready to merge - it's already been tested in the classic portal. Also included a note that explains how it can be done in the EDP (specifying the application_type).

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**